org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Replay/ImportReplay.java

```diff
@@ -541,6 +541,36 @@ public final void testParseSymbologyColor()
       assertEquals("sym type correct", "Hidar", label2.getSymbolType());
       assertEquals("color correct", DebriefColors.PURPLE, label2.getColor());
     }
+    
+    
+    public final void testPasteRep() {
+      String textToPaste =
```

Comment:
@ayeshare - I think it would be sensible to add a few track positions (you could take a few from the start of `boat1.rep`).  You should then check that the new Track is in present in the `Layers` object, and that it contains the correct number of points.